contrib/auto-completion/packages.el

```diff
@@ -193,18 +193,31 @@
       ;; use hippie-expand instead
       (setq yas-minor-mode-map (make-sparse-keymap))
 
+      (setq spacemacs_snippets::dir
```

Comment:
i think @syl20bnr is asking why there's an underscore and a :: when elisp is usually spine-cased